docs/basics/cli-commands-explained.mdx

```diff
@@ -0,0 +1,116 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# CLI commands explained
+
+The Superface CLI is the starting point for every use case you define, this guide presents an overview of all the commands, what they do and what files they create.
+
+If you don't have the CLI installed yet, you can do so with the commands below. You'll need Node.js version 18 or higher to use it.
+
+<Tabs
+  defaultValue="macos"
+  values={[
+    { label: 'macOS', value: 'macos' },
+    { label: 'Linux', value: 'linux' },
+    { label: 'Windows', value: 'windows' },
+  ]
+}>
+
+<TabItem value="macos">
+
+```shell title="Install with Homebrew"
+brew install superfaceai/cli/superface
+```
+
+</TabItem>
+<TabItem value="linux">
+
+```shell title="Install with NPM"
+npm install -g @superfaceai/cli@latest
+```
+
+</TabItem>
+<TabItem value="windows">
+
+```shell title="Install with NPM"
+npm install -g @superfaceai/cli@latest
+```
+
+</TabItem>
+</Tabs>
+
+After installing the CLI, the first command you should run is:
+
+## `superface login`
+
+Authenticates the CLI with your Superface account so that you can store the Comlink profiles you create, as well as track metrics around their usage.
+
+## `superface prepare`
+
+The `prepare` command is used with either a URL or file path to an [Open API Specification](https://spec.openapis.org/oas/latest.html) document for the API you want to work with.
```

Comment:
This is saying how to use it, not what it does (index the API documentation for further processing by Superface AI)

Reply:
**"index API documentation for further AI processing, creates Comlink provider"**